docs/UsingNetlifyGuide.md

```diff
@@ -0,0 +1,45 @@
+## Using RepoSense with Netlify
+
+[Netlify](https://www.netlify.com) enables you to run RepoSense directly on your target repository. You can do everything on the browser itself and Netlify will handle running RepoSense for you. Therefore, as a user you don't need to download the latest jar file every time when wanting to run RepoSense.
+
+1. Fork the RepoSense repository using this [link](https://github.com/repoSense/RepoSense/fork).
+1. Set up Netlify in your forked repository. We recommend this [site](https://www.netlify.com/blog/2016/09/29/a-step-by-step-guide-deploying-on-netlify/) for ease of setup. Please refer to the fields below for **Step 5**.
+   
+   **Build command**:
+   ```
+   ./gradlew run -Dargs="--repos YOUR_TARGET_REPO --output output_path/ --formats java adoc js"
+   ```
+   **Publish directory**:
+   ```
+   /output_path/reposense-report
+   ```
+1. Click on **Settings** in the top, choose **Build & deploy** from the left panel and scroll to **Build hooks**.
```

Comment:
We don't need webhooks to get first look at the report right?

Then, we can ask the users to check if report is being generated and let them view the report first. Then we can group this into another section.

A shorter initial setup is more compelling for users.

Reply:
Okay!

Reply:
Okay sure.